.tx/config

```diff
@@ -175,3 +181,8 @@ type = KEYVALUEJSON
 source_file = src/views/ideas/l10n.json
 source_lang = en
 type = KEYVALUEJSON
+
```

Comment:
These lines below look like they're duplicated unnecessarily

Reply:
Yup, too many copy/pastes. I force pushed to keep the commit history cleaner.